### DISCUSSION TITLE
What if WhatsApp adds some backward incompatible changes, Do i have to donate again?

### DISCUSSION THREAD

**Discussion by ExileD-dev** [devlikeapro/waha#63] [Q&A]
What if WhatsApp adds some backward incompatible changes right after I donate, or after a week, do I have to donate again in order to use WAHA?

**Comment by allburov** [comment]
Yes, sometimes it happens. From my observation 1 time every 3-6 months, depends on functions that you use.

**Reply by ExileD-dev** [reply]
so when was the last time they updated?

**Comment by allburov** [comment]
I'm talking only about Plus version, btw, Core one is always available

**Comment by devlikepro** [comment]
https://waha.devlike.pro/docs/overview/faq/#i-want-to-purchase-waha-plus-waha-pro-is-it-one-time-purchase

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
